### PR TITLE
Fix some issues in radius directory

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/radius/functions/radius-cvt-int.xml [hunk 1]
```diff
@@ -14,7 +14,30 @@
...
+ </refsect1>
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  <para>
+   <variablelist>
+    <varlistentry>
+     <term><parameter>data</parameter></term>
+     <listitem>
+      <para>
```
Comment (Girgias) [inline reference/radius/functions/radius-cvt-int.xml:29]: Just add something like `Input data`

File: reference/radius/functions/radius-cvt-string.xml [hunk 2]
```diff
@@ -14,10 +14,32 @@
...
 
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  <para>
+   <variablelist>
+    <varlistentry>
+     <term><parameter>data</parameter></term>
+     <listitem>
+      <para>
+
```
Comment (Girgias) [inline reference/radius/functions/radius-cvt-string.xml:29]: Ditto

File: reference/radius/functions/radius-demangle.xml [hunk 3]
```diff
@@ -20,6 +20,23 @@
...
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  <para>
+   <variablelist>
+    &radius.parameter.handle;
+    <varlistentry>
+     <term><parameter>mangled</parameter></term>
+     <listitem>
+      <para>
+
```
Comment (Girgias) [inline reference/radius/functions/radius-demangle.xml:32]: Maybe `The mangled data to demangle` ?

**Comment by saundefined** [comment]
@Girgias, fixed, thank you!

**Review by Girgias — APPROVED** [review]

**Pull request merged**